packages/asset/contracts/Asset.sol

```diff
@@ -30,6 +30,9 @@ import {TokenIdUtils} from "./libraries/TokenIdUtils.sol";
 import {IAsset} from "./interfaces/IAsset.sol";
 import {ITokenUtils, IRoyaltyUGC} from "./interfaces/ITokenUtils.sol";
 
+/// @title Asset
+/// @author TheSandbox
+/// @notice Base asset contract for ERC1155 tokens
```

Comment:
What do you mean by that?

Would it sound better if I rename it to:
```
/// @notice Core ERC1155 asset contract
/// @notice Minting and burning tokens is only allowed through separate authorized contracts
```

You see what I'm trying to explain hopefully but I'd appreciate if you can help me phrase it better.

Reply:
I would do something like this:
```
/// @notice ERC1155 asset token contract
/// @notice Minting and burning tokens is only allowed through separate authorized contracts
/// @dev This contract is final and should not be inherited
```
(Usually when we denote something as a base contract, it means we can inherit it in other contracts. However, we don't want Asset.sol to be inherited, since it is also final)